src/urllib3/filepost.py

```diff
@@ -17,8 +14,7 @@ def choose_boundary():
     Our embarrassingly-simple replacement for mimetools.choose_boundary.
     """
     boundary = binascii.hexlify(os.urandom(16))
-    if not six.PY2:
-        boundary = boundary.decode("ascii")
+    boundary = boundary.decode("ascii")
```

Comment:
Can we combine all these lines into a single `return` statement? (also `.decode()` instead of `.decode("ascii")`)